app/src/main/java/com/xmartlabs/template/log/logger/CrashlyticsLogger.java

```diff
@@ -0,0 +1,21 @@
+package com.xmartlabs.template.log.logger;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.crashlytics.android.Crashlytics;
+import com.xmartlabs.template.log.LogInfo;
+import com.xmartlabs.template.log.Logger;
+
+/** {@link Logger} that logs exceptions/events to Crashlytics **/
+public class CrashlyticsLogger implements Logger {
+  @Override
+  public void log(@NonNull LogInfo logInfo, @Nullable Throwable t) {
+    Crashlytics.log(logInfo.getPriority(), logInfo.getTag(), logInfo.getMessage());
```

Comment:
It will only log the message twice if no exception if present, but that's necessary because logs are only sent to crashlytics when a crash occurs or an exception is logged